.circleci/config.yml

```diff
@@ -374,9 +374,9 @@ jobs:
       - run:
           name: Use py37
           command: |
-            pyenv install -s 3.7.0
+            pyenv install -s 3.7.9
```

Comment:
there are even later version of py37 if we want to update

Reply:
well, I thought, something not too old and not too recent would be a good compromise.